Python/crossfire/tests/test_occurences.py

```diff
@@ -43,3 +43,21 @@ def test_occurrences_stops_when_there_are_no_more_pages():
     occurrences = tuple(occurence for occurence in Occurrences(client, id_state="42"))
     assert client.get.call_count == 2
     assert len(occurrences) == 4
+
+
+def test_occurrences_with_obligtory_parameters():
+    client = Mock()
+    client.get.return_value = dummy_response(True)
+    client.URL = "https://api-service.fogocruzado.org.br/api/v2/"
```

Comment:
Acho legal não usar a URL de verdade aqui. Não queremos o teste utilizando a API real em nenhuma hipótese, então, caso errarmos o _mock_ (não é o caso aqui), o melhor seria ter uma URL de mentirinha:


```suggestion
    client.URL = "https://127.0.0.1/"
```

(mesma coisa na linha 59)

Reply:
Perfeito.
Alterado no ultimo commit.